### PR TITLE
refactor(core/utils): remove makeOwnerSwapper()

### PR DESCRIPTION
**Pull request by Swapnilr1**
Fixed issue #1959 (Remove makeOwnerSwapper())

### CODE REVIEW

**Comment by Swapnilr1** [comment]
@marcoscaceres Thanks, I have removed those changes. Please review it again.
@sidvishnoi Thanks for changing the title.

File: src/core/utils.js [hunk 1]
```diff
@@ -115,16 +115,6 @@ export const nonNormativeSelector =
  *                    insertion point (Node) - or just appended, if
  *                    the element has no children.
  */
-export function makeOwnerSwapper(node) {
```
Comment (sidvishnoi) [inline src/core/utils.js]: Missed the comments for this function above 😅
Rest is all good 👌

Comment (Swapnilr1) [reply src/core/utils.js]: @sidvishnoi Yes, I did not realize those comments were for this function. Can you please check now?

**Comment by marcoscaceres** [comment]
Thanks @Swapnilr1! Let us know if you want to work on another.

**Pull request merged**